### ISSUE TITLE
All menu headers, titles and options say "help us" instead of the right words. 

### ISSUE THREAD

**Issue by ballistic56** [darkdks/KF2ServerTool#27]
Problem first started when updating to 1.37 I believe, or before that. I noticed in the KF2ServerTool that all the words just say "help us". This is all menu items and headers. The fields you have to fill are fine. The tool still works and I am able to launch my server, update maps and game files but I have to click on buttons sometimes until I find the right buttons. Obviously, not ideal.
 I have included a link to a screenshot to show the problem. 

When I first start the tool I get a message, "Failed to set language. Language with initial "EG" not found". I am not 100% certain if that message is related but I know I had that message before when the tool worked. I just wanted to include it just in case. 

**To Reproduce**
1. Launch KF2ServerTool
2. Initial error message stating "Failed to set language..." See above
3. Click OK to clear the message. 
4. KF2ServerTool starts. 

**Expected behavior**
The expected behavior is for the menu headers, titles of options etc to have the normal words instead of "help us". 

**Screenshots**
Here is the [link](https://ibb.co/fpdhWnr)
![KFServerTool_ISSUE](https://user-images.githubusercontent.com/77619161/104913719-09ba4e00-595c-11eb-8a22-77bd8d4df9a7.JPG)

Backup - https://ibb.co/fpdhWnr

**Desktop (please complete the following information):**
 - OS: Windows 10 64bits
 - Version 1.4

**Comment by ballistic56** [comment]
I noticed a file called "KF2ServerTool_MissingLocalization.txt" inside C:\kf2server. If I go into C:\kf2server\KFGame\Localization there is no EG file. So, maybe this is not a bug but just a missing file. I have tried removing the files and adding back the KF2ServerTool files but it still had the same problem which is frustrating. Maybe I missed something?

That text document has the following inside of it, 
"Help us"="Help us"
"Reinstall an item, helpful to fix a broken item or add some entries"="Reinstall an item, helpful to fix a broken item or add some entries"
"Example: "="Example: "
"Map File"="Map File"
"Subscription"="Subscription"
"Map Entry"="Map Entry"
"Cycle Entry"="Cycle Entry"
"Map Cache"="Map Cache"
"Mod File"="Mod File"
"In Cache"="In Cache"
"Normal"="Normal"
"Hard"="Hard"
"Suicidal"="Suicidal"
"Hell on earth"="Hell on earth"
"Short"="Short"
"Medium"="Medium"
"Long"="Long"
"Disabled"="Disabled"
"Enabled"="Enabled"
"Simple report List"="Simple report List"
"Thumbnail mode"="Thumbnail mode"
"Icon mode"="Icon mode"
"New server..."="New server..."
